documentation/docusaurus.config.js

```diff
@@ -71,18 +64,18 @@ async function createConfig() {
           },
           items: [
             {
-              type: 'doc',
-              docId: 'introduction',
+              type: 'docSidebar',
+              sidebarId: 'about',
               position: 'left',
               label: 'Docs',
```

Comment:
The mismatch between label, sidebarId and folder name could be confusing

Reply:
@ingeridhellen  Not sure how to best solve this, either we need to have /docs/docs and Docs or use /docs/about and About. Any input?

Reply:
we do this another PR if required